pos_ask_vat/static/src/js/Screens/OrderManagementControlPanel.js

```diff
@@ -0,0 +1,27 @@
+odoo.define("pos_ask_vat.OrderManagementControlPanel", function (require) {
+    "use strict";
+
+    const Registries = require("point_of_sale.Registries");
+    const OrderManagementControlPanel = require("point_of_sale.OrderManagementControlPanel");
+
+    const SEARCH_FIELDS = [
+        "pos_reference",
+        "partner_id.display_name",
+        "date_order",
+        "customer_tax_id",
+    ];
+
+    const OrderManagementControlPanelCRM = (OrderManagementControlPanel) =>
+        class extends OrderManagementControlPanel {
+            get searchFields() {
+                return SEARCH_FIELDS;
```

Comment:
I think it would be better to get the list from calling super and then append "customer_tax_id" to it. Otherwise it would conflict if another module also tries to extends the search fields.